examples/settings.py

```diff
@@ -3,11 +3,31 @@
 # LICENSE file in the root directory of this source tree.
 
 import habitat_sim
-from habitat_sim.utils.settings import default_sim_settings
-from habitat_sim.utils.settings import make_cfg as _make_cfg
+import habitat_sim.agent
 
-example_settings = {
+default_sim_settings = {
+    # settings shared by example.py and benchmark.py
     "max_frames": 1000,
+    "width": 640,
+    "height": 480,
+    "default_agent": 0,
```

Comment:
What is this config for?

Reply:
`default_agent` is used in `viewer.py` and `width`/`height` are used in other scripts in the `examples` directory